MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -433,10 +501,27 @@ - (void)acquireTokenInteractive:(id)sender
                 [self updateResultViewError:error];
             }
             
+            [_webView loadHTMLString:@"<html><head></head><body>done!</body></html>" baseURL:nil];
+            [_authView setHidden:YES];
+            [self.view setNeedsDisplay];
+            
             [[NSNotificationCenter defaultCenter] postNotificationName:MSALTestAppCacheChangeNotification object:self];
         });
     };
     
+    application.webviewSelection = _webviewSelection.selectedSegmentIndex == 0 ? MSALWebviewSelectionEmbedded : MSALWebviewSelectionSystemDefault;
+    
+    if (application.webviewSelection == MSALWebviewSelectionEmbedded &&
+        [[_customWebViewSelection titleForSegmentAtIndex:_customWebViewSelection.selectedSegmentIndex] isEqualToString:TEST_EMBEDDED_CUSTOM])
+    {
+        application.customWebview = _webView;
+        
+        [UIView animateWithDuration:0.5 animations:^{
+            [_acquireSettingsView setHidden:YES];
```

Comment:
Is hidden animatable :)?

Reply:
hm, i don't think so :p set hidden is not changing alpha or anything.